data-raw/ebola1976.R

```diff
@@ -0,0 +1,12 @@
+## code to prepare `ebola1976` dataset goes here
+
+ebola1976 <- read.csv(
```

Comment:
As long as we don't know the source of this dataset, this doesn't bring any value. We can remove this script and the csv from `extdata`

Reply:
I've asked @adamkucharski about the data source, and there is a source which I shall add here.

Reply:
Data source added as Camacho et al. https://doi.org/10.1016/j.epidem.2014.09.003